5.5/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:centos7
+
```

Comment:
@bparees +1 we should put as many comments into Dockerfile(s) as we can. Especially to describe our reasoning of why we did the things in this way, to avoid future confusion.